WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderViewModel.kt

```diff
@@ -31,7 +32,8 @@ class ReaderViewModel @Inject constructor(
     @Named(BG_THREAD) private val bgDispatcher: CoroutineDispatcher,
     private val appPrefsWrapper: AppPrefsWrapper,
     private val dateProvider: DateProvider,
-    private val loadReaderTabsUseCase: LoadReaderTabsUseCase
+    private val loadReaderTabsUseCase: LoadReaderTabsUseCase,
+    private val resourceProvider: ResourceProvider
```

Comment:
I believe we can remove the resourceProvider now, right? It should fix the failing tests.

Reply:
> Agreed, I already did, it's in the next commit with the updated unit tests. Wanted opinion on default fun first.

Reply:
Ohh I see. I usually assume when someone uses a top-level comment (such as this one https://github.com/wordpress-mobile/WordPress-Android/pull/11929#issuecomment-630794673) that the changes are ready for another round. My bad, I should have waited for an explicit comment ;).

Reply:
My bad. Apologies.

Reply:
Really no reason to apologize :). You didn't do anything wrong.